includes/User.php

```diff
@@ -1656,8 +1656,9 @@ public function pingLimiter( $action = 'edit' ) {
 	 *
 	 * @return Bool True if blocked, false otherwise
 	 */
-	public function isBlocked( $bFromSlave = true, $shouldLogBlockInStats = true ) { // hacked from false due to horrible probs on site
-		return $this->getBlock( $bFromSlave, $shouldLogBlockInStats ) instanceof Block && $this->getBlock()->prevents( 'edit' );
+	public function isBlocked( $bFromSlave = true, $shouldLogBlockInStats = true, $global = true ) { // hacked from false due to horrible probs on site
```

Comment:
:speech_balloon: — for backwards compatibility, assume we are dealing with a global block;